assets/css/style.css

```diff
@@ -49,8 +87,24 @@ button {
   white-space: nowrap;
 }
 
-.promo-section {
-  padding-bottom: 130px;
+.content-title {
+  color: rgb(255, 255, 255);
+  font-family: 'Yeseva One', 'Arial', 'Times New Roman', serif;
+  font-size: 90px;
+  font-weight: 100;
+  max-width: 970px;
+  text-align: center;
+  margin: 0px auto;
+}
+
+div {
+  box-sizing: border-box;
+}
```

Comment:
It is defined here - https://github.com/thekarinaten/lambda-restaurant/pull/5/files#diff-b8392010d5f6b5c8a2f493b3a11e3109R54